src/Raven.Studio/wwwroot/App/views/shell.html

```diff
@@ -1,27 +1,5 @@
-<div class="browser-alert" data-bind="visible: browserAlert">
-    <div class="browser-message">
-        <i class="icon-unsupported-browser icon-xl"></i>
-        <p class="lead margin-bottom">This browser is not supported and some features may not work correctly</p>
-        <p class="margin-bottom">To ensure best experience please use</p>
-        <div class="browser-links">
-            <a class="chrome" href="https://www.google.com/chrome/" target="_blank">
-                <i class="icon-chrome icon-lg"></i>
-                Google Chrome
-            </a>
-            <a class="firefox" href="https://www.mozilla.org/firefox/" target="_blank">
-                <i class="icon-firefox icon-lg"></i>
-                Mozilla Firefox
-            </a>
-        </div>
-        <button class="btn btn-default" data-bind="click: browserAlertContinue">Continue</button>
-        <div class="checkbox">
-            <input id="dontShowAgainBrowser" class="styled" type="checkbox" data-bind="checked: dontShowBrowserAlertAgain">
-            <label for="dontShowAgainBrowser">
-                Don't show again
-            </label>
-        </div>
-    </div>
-    <div class="backdrop show"></div>
+<div data-bind="visible: browserAlert()">
```

Comment:
can we move that logic inside and compose viewmodel instead of html file?

Reply:
done